scripts/dats_validator/README.md

```diff
@@ -0,0 +1,20 @@
+# Simple dats validator
+
+**Set up:**
+
+- clone the repo, **important:** use `--recursive` to fetch the latest version of dats submodule:
+<pre>git clone https://github.com/zxenia/dats_validator.git --recursive</pre>
```

Comment:
This should be updated as I don't think it's valid for a script in conp-dataset.

Reply:
Thanks for the review I'll fix it now